samples/excel/50-workbook/create-workbook.yaml

```diff
@@ -0,0 +1,86 @@
+order: 2
+id: excel-workbook-create-workbook
+name: Create workbook
+description: Sample for Excel
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.8
+script:
+    content: |
+        $("#file").change(() => tryCatch(createWorkbook));
+
+        async function createWorkbook() {
+            const myFile = document.getElementById("file");
+            const reader = new FileReader();
+
+            reader.onload = ((theFile) => {
```

Comment:
Why are we assigning an IFFE to reader.onload? Examples I find online for reader.onload don't do this. Maybe I'm missing something, but this looks like its going to pointlessly call the function immediately after it's assigned to onload, even before the readAsDataURL line runs, and then run again when the latter line runs.